src/main/java/tutoraid/model/ModelManager.java

```diff
@@ -22,8 +24,8 @@
  * Represents the in-memory model of the student book data.
  */
 public class ModelManager implements Model {
+    private static final List<Student> allStudents = new ArrayList<>();
```

Comment:
Update this ArrayList of Students so that Lesson objects can refer to it if needed to see which Students belong to it